pkgs/os-specific/linux/facetimehd/default.nix

```diff
@@ -17,8 +17,8 @@ stdenv.mkDerivation rec {
   src = fetchFromGitHub {
     owner = "patjak";
     repo = "bcwc_pcie";
```

Comment:
I think the repository was renamed
```suggestion
    repo = "facetimehd";
```

Reply:
A long time ago, just been lazy and relying on the redirect ;)